src/marqo/core/index_management/validation.py

```diff
@@ -0,0 +1,29 @@
+from marqo.tensor_search.models.index_settings import IndexSettings
```

Comment:
@farshidz Would it be better to leave this module as separated python file, or should it be included in the index_management class?

Reply:
Sorry @ali60 I meant having this as a function within `IndexManagement` class. I agree with @wanliAlex it doesn't need its own module

Reply:
ok I'll move it

Reply:
btw `IndexManagement` looks like a big class with many responsibilities